include/seqan3/range/view/take.hpp

```diff
@@ -321,12 +321,22 @@ class view_take : public ranges::view_interface<view_take<urng_t, exactly, or_th
     constexpr view_take(urng_t _urange, size_t const _size)
         : urange{std::move(_urange)}, target_size{_size}
```

Comment:
btw, you can give the arguments the same name as the classes variables. Only make sure to prefix the member in the constructor's body with `this->`

Reply:
I hate `this->`

Reply:
I hate the ugly `_` prefix on the constructor arguments.